File: Model/Entry.vb
Public Class Entry

    Property ProjectId As Integer
    Property Id As Integer
    Property Description As String
    Property StartTime As TimeSpan
    Property EndTime As TimeSpan
    Property User As User
    Property CostPerHour As Integer

    Public Shared Function GetEntries() As IEnumerable(Of Entry)

        Return New List(Of Entry) From {
            New Entry With {.ProjectId = 1, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}},
            New Entry With {.ProjectId = 4, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}},
            New Entry With {.ProjectId = 3, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}},
            New Entry With {.ProjectId = 1, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}},
            New Entry With {.ProjectId = 3, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}},
            New Entry With {.ProjectId = 1, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}},
            New Entry With {.ProjectId = 1, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}},
            New Entry With {.ProjectId = 3, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}},
            New Entry With {.ProjectId = 1, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}},
            New Entry With {.ProjectId = 2, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}},
            New Entry With {.ProjectId = 1, .Id = 1, .Description = "Just a Test", .StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks),
                            .EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), .CostPerHour = 45, .User = New User With {
                            .FirstName = "Kyle", .LastName = "Admin", .Email = "k.admin@online.com", .Password = "adminpassword", .IsAdmin = True}}
        }
    End Function

    Public Shared Function GetEntries(Id As Integer) As IEnumerable(Of Entry)
        Dim Entries = GetEntries()

        Return New List(Of Entry)(Entries.Where(Function(c) c.ProjectId = Id))
    End Function

End Class

File: ModelCS/Entry.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;

namespace Model
{
    public class Entry : EntityBase
    {
        [ForeignKey("ProjectId")]
        public Project Project { get; set; }
        public int ProjectId { get; set; }
        public string Description { get; set; }
        public TimeSpan StartTime { get; set; }
        public TimeSpan EndTime { get; set; }
        public int UserId { get; set; }
        [ForeignKey("UserId")]
        public User User { get; set; }
        public int CostPerHour { get; set; }

        //public static IEnumerable<Entry> GetEntries()
        //{
        //    return new List<Entry>()
        //    {
        //        new Entry() { ProjectId = 1, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } },
        //        new Entry() { ProjectId = 4, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } },
        //        new Entry() { ProjectId = 3, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } },
        //        new Entry() { ProjectId = 1, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } },
        //        new Entry() { ProjectId = 3, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } },
        //        new Entry() { ProjectId = 1, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } },
        //        new Entry() { ProjectId = 1, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } },
        //        new Entry() { ProjectId = 3, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } },
        //        new Entry() { ProjectId = 1, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } },
        //        new Entry() { ProjectId = 2, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } },
        //        new Entry() { ProjectId = 1, Id = 1, Description = "Just a Test", StartTime = TimeSpan.FromTicks(DateTime.Now.Ticks), EndTime = TimeSpan.FromTicks(DateTime.Now.Ticks), CostPerHour = 45, User = new User() { FirstName = "Kyle", LastName = "Admin", Email = "k.admin@online.com", Password = "adminpassword", IsAdmin = true } }
        //    };
        //}

        //public static IEnumerable<Entry> GetEntries(int Id)
        //{
        //    return new List<Entry>(GetEntries().Where(e => e.ProjectId == Id));
        //}
    }
}
